### ISSUE TITLE
The requested resource /module/web/NewsAdministration was not found

### ISSUE THREAD

**Issue by zelcode** [georgringer/news#1232]
## Bug Report

**Current Behavior**
After update 7.3.1 -> 8.0.0
```
tx_news.module {
        preselect {
                topNewsRestriction = 1
        }
}
```
If set any `preselect` options, there is error in Administration module:

(1/1) #1425389240 TYPO3\CMS\Backend\Routing\Exception\ResourceNotFoundException
The requested resource "/module/web/NewsAdministration" was not found.

**Expected behavior/output**
No error.

**Environment**
- TYPO3 version(s): 9.5.15
- news version: 7.3.1 -> 8.0.0
- Is your TYPO3 installation set up with Composer (Composer Mode): no
- OS: Linux Mint 19.3

**Possible Solution**

**Comment by janslu** [comment]
Thanks for this report. I've had the same problem for 
```
tx_news.module {
        preselect {
		sortingField = datetime
		sortingDirection = desc
	}
}
```

**Comment by zelcode** [comment]
Closed in favor of #1271
